### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by danilodileo**
## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/metatdenovo/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/metatdenovo _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [x] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [x] `README.md` is updated (including new tool citations and authors/contributors).

I made some small changes in the documentation, you can have a look and add/remove things

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ On release, automated continuous integration tests run the pipeline on a full-si
 2. Present QC for raw reads ([`MultiQC`](http://multiqc.info/))
 3. Quality trimming and adapters removal for raw reads [`Trimm Galore!`](https://www.bioinformatics.babraham.ac.uk/projects/trim_galore/)
 4. Filter sequences with [`BBduk`](https://jgi.doe.gov/data-and-tools/software-tools/bbtools/bb-tools-user-guide/bbduk-guide/)
-5. Normalize the sequences with [khmer](https://khmer-protocols.readthedocs.io/en/latest/mrnaseq/2-diginorm.html) or [`BBnorm`](https://jgi.doe.gov/data-and-tools/software-tools/bbtools/bb-tools-user-guide/bbnorm-guide/)
+5. Normalize the sequences with [`BBnorm`](https://jgi.doe.gov/data-and-tools/software-tools/bbtools/bb-tools-user-guide/bbnorm-guide/)
```
Comment (erikrikarddaniel) [inline README.md]:
```suggestion
5. Normalize the sequencing depth with [`BBnorm`](https://jgi.doe.gov/data-and-tools/software-tools/bbtools/bb-tools-user-guide/bbnorm-guide/)
```

File: README.md [hunk 2]
```diff
@@ -47,7 +47,6 @@ On release, automated continuous integration tests run the pipeline on a full-si
 11. Taxonomical annotation:
```
Comment (erikrikarddaniel) [inline README.md]:
```suggestion
11. Taxonomic annotation:
```

File: docs/usage.md [hunk 3]
```diff
@@ -61,15 +61,11 @@ For further documentation, see the [BBduk official website](https://jgi.doe.gov/
...
-
-- Khmer_based approach (`--diginorm`)
+There is one option for digital normalization in the pipeline:
 
 - bbnorm (`--bbnorm`)
 
-you can run only one option per assembly.
-
-> Please, check the [khmer](https://khmer-protocols.readthedocs.io/en/v0.8.4/) and the [bbnorm](https://jgi.doe.gov/data-and-tools/software-tools/bbtools/bb-tools-user-guide/bbnorm-guide/) documentation for further information about these programs and how digital normalization works. Remember to check [Parameters](https://nf-co.re/metatdenovo/parameters) page for the full option that can be used for this step
+> Please, check the [bbnorm](https://jgi.doe.gov/data-and-tools/software-tools/bbtools/bb-tools-user-guide/bbnorm-guide/) documentation for further information about these programs and how digital normalization works. Remember to check [Parameters](https://nf-co.re/metatdenovo/parameters) page for the full option that can be used for this step
```
Comment (erikrikarddaniel) [inline docs/usage.md]:
```suggestion
> Please, check the [bbnorm](https://jgi.doe.gov/data-and-tools/software-tools/bbtools/bb-tools-user-guide/bbnorm-guide/) documentation for further information about these programs and how digital normalization works. Remember to check [Parameters](https://nf-co.re/metatdenovo/parameters) page for the all options that can be used for this step.
```

File: docs/usage.md [hunk 4]
```diff
@@ -129,9 +125,7 @@ These options are:
 
 - [kofamscan](https://github.com/takaram/kofam_scan) (`--kofam_dir`)
 
-- [EUKulele](https://github.com/AlexanderLabWHOI/EUKulele) (`--eukulele_dbpath`)
-
-All the options can run in the same time (e.g. `nextflow run main.nf -profile test,docker --eggnog --hmmdir hmms/ --rundbcan`) but each program has its own options that you will need to read carefully before running the pipeline.
+All the options can run in the same time (e.g. `nextflow run main.nf -profile test,docker --eggnog --hmmdir hmms/ `) but each program has its own options that you will need to read carefully before running the pipeline.
```
Comment (erikrikarddaniel) [inline docs/usage.md]:
```suggestion
All the options can run at the same time (e.g. `nextflow run main.nf -profile test,docker --eggnog --hmmdir hmms/ `) but each program has its own options that you will need to read carefully before running the pipeline.
```

**Review by erikrikarddaniel — APPROVED** [review]
Looks fine. As we've talked about, we need a session where we sit down together, read and edit.

File: README.md [hunk 5]
```diff
@@ -1,4 +1,4 @@
-# ![nf-core/metatdenovo](docs/images/nf-core-metatdenovo_logo_light.png#gh-light-mode-only) ![nf-core/metatdenovo](docs/images/nf-core-metatdenovo_logo_dark.png#gh-dark-mode-only)
+# ![nf-core/metatdenovo](docs/images/nf-core-metatdenovo_logo_light.png#gh-light-mode-only)
```
Comment (mashehu) [inline README.md]:
```suggestion
<picture>
  <source media="(prefers-color-scheme: dark)" srcset="docs/images/nf-core-metatdenovo_logo_dark.png">
  <img alt="nf-core/metadenovo logo" src="docs/images/nf-core-metatdenovo_logo_light.png">
</picture>
```

File: docs/usage.md [hunk 6]
```diff
@@ -141,9 +135,9 @@ If you don't want run eggNOG-mapper, you will need to add the flag `--skip_eggno
 
 ## Example pipeline command with some common features
 
-`nextflow run lnuc-eemis/metatdenovo -profile docker --input samplesheet.csv --assembler rnaspades --orf_caller transdecoder --eggnog --run_dbcan`
+`nextflow run lnuc-eemis/metatdenovo -profile docker --input samplesheet.csv --assembler rnaspades --orf_caller transdecoder --eggnog`
```
Comment (mashehu) [inline docs/usage.md]:
```suggestion
```nextflow
nextflow run lnuc-eemis/metatdenovo -profile docker --input samplesheet.csv --assembler rnaspades --orf_caller transdecoder --eggnog
```
```
By having it as a code block with syntax highlighting you get also a copy-button rendered along with it on the website

**Review by mashehu — APPROVED** [review]

**Pull request merged**